Automattic/jetpack
# Stats: clear dashboard API cache on purchase or force refresh

## Proposed changes:

Stop caching API for dashboard management after purchase or on `force_refresh`.

For robustness, we should pass `force_refresh` from Calypso API calls as well because `referer` header might not be available in some scenarios, which will be addressed later.

### Other information:


## Jetpack product discussion
n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:

* Purchase Stats product for a Jetpack site thru Odyssey `/wp-admin/admin.php?page=stats#!/stats/purchase/:site`
* Ensure the subscription is recognized right after purchase success back to the dashboard
  * If it's commercial, then please scroll down to the usage bar section, and ensure the view limit is up to date
